schemas/2015-06-01/Microsoft.KeyVault.json

```diff
@@ -0,0 +1,209 @@
+{
+    "id": "http://schema.management.azure.com/schemas/2015-06-01/Microsoft.KeyVault.json#",
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "title": "Microsoft.KeyVault",
+    "description": "Microsoft KeyVault Resource Types",
+    "definitions": {
+        "keyPermission": {
+            "anyOf": [
```

Comment:
Thanks.  I'll change this to be a simple array that accepts exactly what the provider accepts as valid inputs, then.